workspaces/ui-v2/src/contexts/SpecRepositoryContext.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from 'react';
+import { useContext } from 'react';
+import { IOpticSpecRepository } from '@useoptic/spectacle';
+
+export function useSpecRepository(): {
+  specRepo: IOpticSpecRepository;
+} {
+  const specRepo = useContext(SpecRepositoryContext)!;
```

Comment:
Given we are creating the context with a default of null - null will be returned if there isn't a parent provider. Can we do a `if !specRepo -> throw error` - similar to https://github.com/opticdev/optic/blob/develop/workspaces/ui-v2/src/contexts/spectacle-provider.tsx#L23-L25

Also we could just return `specRepo` straight instead of wrapping it in an object (unlikely we'd want to return anything other than the context when the hook only reads from a single context value

Reply:
Done, and done